io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+app-id: io.github.pemsley.coot
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: coot
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
```

Comment:
why does it need the broad host filesystem access?

Reply:
I think it does not need.

Reply:
Coot is a molecular model editor (like, say, Inkscape, but for proteins). Typically, we will not know where the user has stored their models and crystallographic data. It is often used in conjunction with CCP4i2: http://ccp4i2.fg.oisin.rc-harwell.ac.uk/ - and it would be convenient to be able to read the database of that project to help navigate to the files that the user wishes to use.

Reply:
@permsley

`--filesystem=home` is not enough?

Reply:
That gives us access to our files, but other users? Yes, that will be fine.

Reply:
> That gives us access to our files, but not other users? Yes, that will be fine.

I think so. `home` in `--filesystem=home` means current user's home dir.

Reply:
@pemsley 
> (but not reading the CCP4i2 database, and the files references therein though

Where are these files placed?

Reply:
`$HOME/CCP4I2_PROJECTS` and `$HOME/CCP4I2_DATABASE`

Reply:
OK, let it be `--filesystem=home`.